Readme/automation.md

```diff
@@ -0,0 +1,155 @@
+2. In your TeamCity project, create a `Build configurations` to use the forked repository. If you don't know how to do it, please follow [TeamCity document](https://confluence.jetbrains.com/display/TCD18/Build+Configuration+Template) for instructions.
+3. Clone the forked repository to configure in the next steps.
+## B. SETUP
+### 1. Getting desired capabilities
+Desired capabilities need to be added to the automation test script to allow the test to be executed on Kobiton device. The desiredCap are just matching only for one device. If you want to test on other devices, please make sure noted the different and customize as your need. 
+
+In the provided sample automation test scripts, we have pre-configured them to execute the provided sample application(s) on one of the available device(s) that has the name `Galaxy` for `Android` and `iPhone` for iOS. If you want to use other specific device(s), follow the instruction below to get the corresponding desired capabilities for that device(s).
+
+**How to get desired capabilities**
+
+In this tutorial, we will be showing how to get desired capabilities for executing automation test on `Pixel 2 XL 8.1.0`.
+  1. Go to https://portal.kobiton.com and login to your Kobiton account.
+  2. Click **Devices** in the top navigation bar.
+
+  ![click devices](./assets/devices.png)
+ 
+  3. Hover over any device you want to test with and click on Automation settings button (the one with gear symbol).
+ 
+  ![click auto](./assets/click_auto.png)
+ 
+  4. On the left-hand side, you can select your preferred language, App Type, etc.
+  - In this example, we use **NodeJS** as the default language of the script. Therefore, choose **NodeJS** in `Language` section.
+  - Select **Hybrid/Native from Apps** in `App type` section. 
+
+ Kobiton automatically generates your desired capabilities into code based on the selected options. Note values of `deviceName`, `platformversion` and `platformName` to prepare next steps.
+
+  ![desiredcaps](./assets/desiredcaps.png) 
+### 2. Configuring automation test script
+In the `automation-script` folder, we have provided a sample scripts for executing automation test on Kobiton devices.
+As mentioned above, the desired capabilities in provided automation test scripts have already been pre-configured. If you have chosen custom desired capabilities, please follow steps below to apply the collected desired capabilities.
+
++ Add there environment variables to TeamCity project:
+  + KOBITON_DEVICE_NAME: Kobiton device name ( `deviceName` in the generated desired capabilities)
+  + KOBITON_DEVICE_PLATFORM_NAME: Kobiton device platform name ( `platformName` in the generated desired capabilities)
+  + KOBITON_DEVICE_PLATFORM_VERSION: Kobiton device platform version ( `platformversion` in the generated desired capabilities)
+
+![env](./assets/env_designCap.png)
+ 
+For example, if desired capabilities for executing automation test on Pixel 2 XL running Android 8.1.0
```

Comment:
You just only show a partial example. Please fill it with the rest (refer to mine).